bats/fb-verify-selinux.bats

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bats
+# vim: ft=sh:sw=2:et
+
+set -o pipefail
+
+load os_helper
+
+@test "ensure no SELinux denials" {
+  if tIsRedHatCompatible; then
```

Comment:
Should this also check for `selinuxenabled`?

Reply:
it could, I guess, but we're not disabling selinux in our ci?